Configure.cmake

```diff
@@ -9,6 +9,23 @@ find_library(LIBGMP gmp)
 find_library(LIBRT rt)
 find_library(LIBFFTW3 fftw3)
 
+if (NOT CMAKE_CROSSCOMPILING AND NOT SLEEF_FORCE_FIND_PACKAGE_SSL)
+  find_package(OpenSSL)
+else()
+  # find_package cannot find OpenSSL when cross-compiling
```

Comment:
Why not use directly libssl and libcrypto also when not cross compiling? I suggested OpenSSL just as an example, I didn't want to tie you to a particular implementation.

If possible, simplify this check by removing the OpenSSL branch.

Reply:
find_library cannot find OpenSSL with MSVC. So, I need both.